web-app/app/tricorder/Configure.tsx

```diff
@@ -1,52 +1,53 @@
-import { useEffect } from "react";
-import { buttonClass } from "./classes";
+export type Config = {
```

Comment:
Configure owns the Config type, but it no longer owns the state of config. That's up to the implementing component.